solve issues & add Firebase Crashlytics

alsaidmandor-eng opened the pull request:
.  add Firebase Crashlytics in project.
. add Firebase Crashlytics in feed repository to listen issue. 😈😈
. solve issue null check operator in category item widget by add ( ? ) and remove ( ! ) , in image link and title . 😈😈
. solve issue null check operator in search item  by add ( ? ) and remove ( ! ) , in image link and title .😈😈
 . add modify parameter in search item .

alsaidmandor reviewed (approved):
Good solutions for issues 👍♥️🦅🦅

alsaidmandor committed:
.  add FirebaseCrashlytics in project.
. add FirebaseCrashlytics in feed repository to listen issue.
. solve issue null check operator in category item widget by add ( ? ) and remove ( ! ) , in image link and title .
. solve issue null check operator in search item  by add ( ? ) and remove ( ! ) , in image link and title .
. add modify parameter in search item .
